integration-tests/equations/test_forced_advection.py

```diff
@@ -0,0 +1,88 @@
+"""
+Tests discretisations of the forced advection equation. This checks the
```

Comment:
Maybe we could mention what the test case is? e.g. something like: "This test describes transport of water vapour which will be converted to rain if it exceeds a prescribed saturation profile. The initial condition and saturation profile are chosen to give an analytic solution. The test then checks ..."